webapp/channels/src/components/admin_console/license_settings/team_edition/team_edition_right_panel.tsx

```diff
@@ -77,21 +76,24 @@ const TeamEditionRightPanel: React.FC<TeamEditionRightPanelProps> = ({
                     </button>
                 </p>
                 <p className='upgrade-legal-terms'>
-                    <FormattedMarkdownMessage
-                        id='admin.license.enterprise.upgrade.acceptTermsInitial'
-                        defaultMessage='By clicking **Upgrade**, I agree to the terms of the Mattermost '
+                    <FormattedMessage
+                        id='teamEditionRightPanel.acceptTermsInitial'
```

Comment:
can we have the name space as `admin.licenseSettings.teamEdition.teamEditionRightPanel.xxx` in others as well